generators/python/src/fern_python/generators/pydantic_model/custom_config.py

```diff
@@ -24,3 +24,7 @@ class PydanticModelCustomConfig(BasePydanticModelCustomConfig):
     use_str_enums: bool = True
     # Skip validation of fields (automatically includes additional fields)
     skip_validation: bool = False
+
+
+class FastApiPydanticModelCustomConfig(BasePydanticModelCustomConfig):
```

Comment:
Mind putting this in the fastapi config file? Just so all the fastapi config is in one place (it's also how we do it with SDKs today)

Reply:
Makes sense! Done